### PR TITLE
Add TypeScript types definition

### PR DESCRIPTION
**Pull request by jayvdb**
Also fix package.json homepage & bugs.url

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,17 +2,18 @@
...
-  "homepage": "https://github.com/cowboy/node-exit",
+  "homepage": "https://github.com/gruntjs/node-exit-x",
   "author": "Grunt Development Team (https://gruntjs.com/development-team)",
   "repository": {
     "type": "git",
     "url": "git://github.com/gruntjs/node-exit-x.git"
   },
   "bugs": {
-    "url": "https://github.com/gruntjs/node-exit/issues"
+    "url": "https://github.com/gruntjs/node-exit-x/issues"
```
Comment (jayvdb) [inline package.json:12]: This depends on this repo enabling the "Issues" tab for this project

Comment (jayvdb) [reply package.json:12]: ping @vladikoff

**Comment by jayvdb** [comment]
Thanks for merging. Can it be released so I can use it? Thanks.

**Comment by vladikoff** [comment]
done!

**Pull request merged**